templates/checkout.php

```diff
@@ -25,11 +25,18 @@
 	$timestamp = time();
 
 	$context_params = array(
-		'plugin_id'         => $fs->get_id(),
-		'plugin_public_key' => $fs->get_public_key(),
-		'plugin_version'    => $fs->get_plugin_version(),
+		'plugin_id'      => $fs->get_id(),
+		'plugin_version' => $fs->get_plugin_version(),
+        'public_key'     => $fs->get_public_key(),
+        'mode'           => 'dashboard',
+        'plan_id'        => $_GET['plan_id'],
+        'trial'          => ( isset( $_GET['trial'] ) ? $_GET['trial'] : false ),
 	);
 
+	if ( isset( $_GET['licenses'] ) ) {
+	    $context_params['licenses'] = $_GET['licenses'];
```

Comment:
@fajardoleo verify the type (should be int), and if not valid, let the checkout to choose the right one.